GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2_TACLS.cfg

```diff
@@ -0,0 +1,1606 @@
+
+
+@PART[ht2_MPLM]:FOR[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
+	@MODULE[ModuleFuelTanks]
+	{
+		TANK
+		{
+			name = CarbonDioxide
+			amount = 0
+			maxAmount = 4000
+		}
+		TANK
+		{
+			name = Oxygen
+			amount = 4000
+			maxAmount = 4000
+		}
+		TANK
+		{
+			name = Waste
+			amount = 0
+			maxAmount = 10
+		}
+		TANK
+		{
+			name = WasteWater
+			amount = 0
+			maxAmount = 60
+		}
+		TANK
+		{
+			name = Food
+			amount = 100
+			maxAmount = 100
+		}
+		TANK
+		{
+			name = Water
+			amount = 75
+			maxAmount = 75
+		}
+		TANK
+		{
+			name = LithiumHydroxide
+			amount = 200
+			maxAmount = 200
+		}
+	}
```

Comment:
Can we parameterize this entire block?  I think all of these should assign the same ratios of ... probably all of the resources?  So there should be just one line setting "TACLSBaseQuantity = 1.75" and then a patch looking for that key and building the MFT TANKs using MM math operations.